File: Linguagem C Exercicios/C-Repetitiva-Para/TABUADA.c
#include <stdio.h>

int main()
{
    int n,i,soma = 0;

    printf("Deseja a tabuada para qual valor? ");
    scanf("%d",&n);

    for(i = 1;i <= 10;i++){
        soma = i*n;
        printf("%d x %d = %d\n",n,i,soma);
    };

    return 0;
}

File: Linguagem C#/Repetitiva/Tabuada.cs
using System;

namespace Tabuada {
    class Program {
        static void Main(string[] args) {

            int n,soma;

            Console.Write("Deseja a tabuada para qual valor? ");
            n = int.Parse(Console.ReadLine());

            for (int i = 1; i <= 10; i++) {
                soma = i * n;
                Console.WriteLine(n+" x "+i+" = "+soma);
            };
        }
    }
}




